.github/docker/Dockerfile.ci-image

```diff
@@ -19,87 +12,8 @@ RUN apt-get update && apt-get install -y \
   rsync \
   && rm -rf /var/lib/apt/lists/*
```

Comment:
do we still need lines 10-13?

Reply:
clang was needed for bindgen, rsync to for the release ci.
We need to add `wget curl build-essential`, they are installed in another pass that is removed.